content/applications/websites/livechat/overview/get_started.rst

```diff
@@ -2,82 +2,200 @@
 
-Managing chat requests
-======================
+Set an online chat name
+-----------------------
 
-Conversations initiated by visitors pop up as a direct message, and are shown in *Discuss*.
-Therefore, inquiries can be answered wherever you are in Odoo.
+Before participating in a live chat, operators should update their :guilabel:`Online Chat Name`.
+This is the name that will be displayed to site visitors in the live chat conversation.
 
-.. image:: get_started/discuss.png
+To update the :guilabel:`Online Chat Name`, click on the user name in the upper right corner of any
+page in the database. Select :guilabel:`My Profile`. Scroll to the :guilabel:`Online Chat Name`
+field and enter the preferred name.
+
+.. image:: get_started/live-chat-my-profile.png
    :align: center
-   :alt: View of the discuss application with a message sent through live chat in Odoo
+   :alt: View of the My Profile option in Odoo.
 
-.. note::
-   Conversations are dispatched based on the current workload of the online operators.
+If an :guilabel:`Online Chat Name` is not set, the name displayed will default to the
+:guilabel:`User Name`.
 
-Leave or join a channel
-=======================
+.. example::
+   A user has their full name as their :guilabel:`User Name`, but they do not want to include their
+   last name when participating in live chat. They would then set their :guilabel:`Online Chat
+   Name` to include only their first name.
+
+  .. image:: get_started/live-chat-online-chat-name.png
+     :align: center
+     :alt: View of user profile in Odoo, emphasizing the Online Chat name field.
+
+Join or leave a channel
+-----------------------
+
+To join a live chat channel, go to the :menuselection:`Live Chat` app and click the
+:guilabel:`JOIN` button on the kanban card for the appropriate channel. Any channel where the user
+is currently active will show a :guilabel:`LEAVE` button. Click this button to disconnect from the
+channel.
+
+.. image:: get_started/live-chat-leave-channel.png
+   :align: center
+   :alt: View of a channel form and the option to join a channel for Odoo Live Chat.
+
+.. important::
+   Operators that do not show any activity In Odoo for more than thirty minutes will be considered
+   as disconnected.
+
+Manage live chat requests
+-------------------------
+
+Conversations initiated by site visitors appear as direct messages in the :guilabel:`Discuss`
+application. New conversations will appear in bold.
+
+.. image:: get_started/live-chat-managing-chat-responses.png
+   :align: center
+   :alt: View of the discuss application with a message sent through live chat in Odoo.
 
-Go to :menuselection:`Website --> Configuration --> Settings`, access your channel under
-*Live Chat*, and *Join Channel* or *Leave Channel*.
+When a user is added as an operator in a live chat channel, they will be able to receive chats from
+website visitors wherever they are in the database. Chat windows will open in the bottom right
+corner of the screen.
```

Comment:
Would this section be better suited up where the *operators* are first mentioned (and explained)? Seems a bit disjointed at this part of the doc. What do you think?